operator/static/jquery.js

```diff
@@ -0,0 +1,68 @@
+$(document).ready(function() {
+
+    const logs = [];
+    pollServer();
+
+    // Poll logs from server every 5 seconds
+    function pollServer() {
+        $.ajax('logs', {
+            type: 'GET',
+            success: res => {
+                res.response.forEach(log => {
+                    if (!logs.includes(log)) {
+                        logs.push(log);
+                        $('.log-window').append(`<p>${log}</p>`);
+                    }
+                });
+            }
+        });
+        setTimeout(pollServer, 5000);
+    }
+
+    // Start button handler
+    $('#start').click(() => {
+        const selectedTeam = $("input:radio[name ='team-select']:checked").val();
+        if (selectedTeam === undefined) {
+            alert('Select a team.');
+            return
+        }
+
+        const selectedMission = $("input:radio[name ='mission-select']:checked").val();
+        if (selectedMission === undefined) {
+            alert('Select a mission.');
+            return
+        }
+
+        $.ajax('mission/start', {
+            data: JSON.stringify({id: selectedTeam, mission: selectedMission}),
+            contentType: 'application/json',
+            type: 'POST',
+            success: res => {
```

Comment:
error handling would be really cool. Like if something goes wrong, just do an `alert()` or something?

Reply:
Implemented